src/utils/aadGroup.ts

```diff
@@ -0,0 +1,59 @@
+import { Group } from "@microsoft/microsoft-graph-types";
+import { AxiosRequestConfig } from "axios";
+import request from "../request";
+import { odata } from "./odata";
+
+const graphResource = 'https://graph.microsoft.com';
+
+const getRequestOptions = (url: string, metadata: 'none' | 'minimal' | 'full'): AxiosRequestConfig => ({
+  url: url,
+  headers: {
+    accept: `application/json;odata.metadata=${metadata}`
+  },
+  responseType: 'json'
+});
+
+export const aadGroup = {
+  /**
+   * Retrieve a single group.
+   * @param id Group ID.
+   */
+  async getGroupById(id: string): Promise<Group> {
+    const requestOptions = getRequestOptions(`${graphResource}/v1.0/groups/${id}`, 'none');
+    
+    try {
+      return await request.get<Group>(requestOptions);
+    }
+    catch(ex) {
+      throw Error(`Group with ID ${id} was not found.`);
```

Comment:
Here we're losing the original exception, which is a shame because it will make it harder to debug the error. We could show a user-friendly message, as long as we include the original message from the API as well. This would also be consistent with how all affected commands work today.

Reply:
True, I'll just remove the user-friendly message. It might be confusing to merge our custom message with the original one.